src/Policies.h

```diff
@@ -50,12 +48,22 @@ class Policy : public AbstractRunnableModule
         return "policy";
     }
 
+    void set_modules_sequence(bool sequence)
+    {
+        _modules_sequence = sequence;
+    }
+
+    void set_tap(Tap *tap)
+    {
+        _tap.push_back(tap);
+    }
+
     void set_input_stream(InputStream *input_stream)
     {
-        _input_stream = input_stream;
+        _input_stream.push_back(input_stream);
```

Comment:
similarly, should be `add_input_stream` now

Reply:
done